spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigUtils.java

```diff
@@ -102,10 +110,6 @@ static String getApplicationNamespace(KubernetesClient client, String namespace,
 
 	}
 
-	public static String getApplicationNamespace(KubernetesClient client, String namespace) {
```

Comment:
move this method a bit up, so that the two deprecated methods stay close to each other.